sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala

```diff
@@ -213,6 +216,12 @@ class SinkProgress protected[sql](
   override def toString: String = prettyJson
 
   private[sql] def jsonValue: JValue = {
-    ("description" -> JString(description))
+    ("description" -> JString(description)) ~
+      ("numOutputRows" -> JInt(numOutputRows))
   }
 }
+
+object SinkProgress {
+  def apply(description: String, numOutputRows: Option[Long]): SinkProgress =
+      new SinkProgress(description, numOutputRows.getOrElse(-1L))
```

Comment:
nit: please use 2-spaces.

Reply:
It would be good to impl indentation rules into the scalastyle-config.xml. Not sure how, but could look into that in future.